test/e2e/log_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package logs
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	testLoggerName  = "default"
+	testLoggerLevel = "debug"
+)
+
+const patternLoggers = "/loggers"
+
+type LoggerInfo struct {
+	Name  string `json:"name"`
+	Level string `json:"level"`
+}
+
+func TestLoggerEndToEnd(t *testing.T) {
+	// Mock HTTP server for Kmesh daemon pod
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
You should use the real Kmesh daemon instead of a mock.

We have installed Kmesh in https://github.com/kmesh-net/kmesh/blob/main/test/e2e/run_test.sh